qa-include/app/emails.php

```diff
@@ -131,6 +131,9 @@ function qa_send_email($params)
 		$mailer->Sender=$params['fromemail'];
 		$mailer->FromName=$params['fromname'];
 		$mailer->AddAddress($params['toemail'], $params['toname']);
+		if(!empty($params['replytoemail'])){
+			$mailer->addReplyTo($params['replytoemail'], $params['replytoname']);
+		}
```

Comment:
The older code is wrong, yes: https://github.com/q2a/question2answer/blob/dev/qa-include/vendor/PHPMailer/class.phpmailer.php
Maybe I should change all of them :joy: